software/ossn.yml

```diff
@@ -1,8 +1,8 @@
 name: OSSN
 website_url: https://www.opensource-socialnetwork.org/
-description: Open Source Social Network (OSSN) is a social networking software written in PHP. It allows you to make a social networking website and helps your members build social relationships, with people who share similar professional or personal interests.
+description: Social networking software that allows you to make a social networking website and helps your members build social relationships, with people who share similar professional or personal interests.
 licenses:
-  - GPL-2.0
+  - ⊘ Proprietary
```

Comment:
I don't think this is accurate, their license seems to be an open-source, OSI-approved license https://opensource.org/license/cal-1-0